public-html/searchbook.php

```diff
@@ -30,7 +30,13 @@
     $menu->render_header();
     $menu->render_menu();
     ?>
-    <div class="container search-div">
+    <div class="container">
+		<div class="main">
+        <div class="header">
+			<h3>Filter by:</h3>
```

Comment:
@pragzii3896  I can see 2 `Filter By` after submitting the form:
![image](https://user-images.githubusercontent.com/26863344/94371611-67106200-0115-11eb-906f-39c121aee304.png)
Can you please fix this?